src/components/LandingPage/RegulatedRails.tsx

```
@@ -0,0 +1,127 @@
+                    under your control.
+                </p>
+
+                <h6 className="mt-3 text-sm">
+                    Verified with Persona: SOC 2, ISO 27001, GDPR. Zero data stored by Peanut.
+                </h6>
+            </div>
+
+            <div className="w-full">
+                <MarqueeWrapper backgroundColor="#FFFFF" direction="right" className="border-none ">
+                    {logos.map((logo, i) => (
+                        <div className="btn btn-purple btn-shadow-primary-4 mx-7 mb-2 flex h-26 w-48 items-center gap-2">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)